Cargo.toml

```diff
@@ -8,14 +8,23 @@ homepage = "https://github.com/rustminded/xtask-wasm"
 repository = "https://github.com/rustminded/xtask-wasm"
 documentation = "https://docs.rs/xtask-wasm"
 
+[features]
+default = ["wasm-opt"]
+wasm-opt = ["binaryen", "binary-install", "platforms", "tempfile"]
```

Comment:
nah you don't need binaryen. Wasm-opt is part of binaryen but it's to big to compile so just drop it for now and offer only to download wasm-opt alone